packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getUnsignedEvmTransaction.ts

```diff
@@ -67,10 +67,14 @@ export const getUnsignedEvmTransaction = async ({
   
   const adapter = assertGetEvmChainAdapter(chainId)
 
-  const fees = await getEvmTxFees({
+  // TODO: Are these the correct params for data and value?
```

Comment:
For native EVM sends yes, for tokens, would have to `getErc20Data` and 0 out `value` (same like e.g `buildSendApiTransaction` does internally)